AWS/legos/aws_list_access_keys/aws_list_access_keys.py

```diff
@@ -0,0 +1,42 @@
+##
+# Copyright (c) 2021 unSkript, Inc
+# All rights reserved.
+##
+from pydantic import BaseModel, Field, SecretStr
+from typing import Dict,List
+import pprint
+
+
+class InputSchema(BaseModel):
+    aws_username: str = Field(
+        title="Username",
+        description="Username of the IAM User"
+    )
+
+
+def aws_list_access_keys_printer(output):
+    if output is None:
+        return
+    pprint.pprint(output)
+
+
+def aws_list_access_keys(
+    handle,
+    aws_username: str
+) -> List:
+    """aws_check_ssl_certificate_expiry checks the expiry date of an ACM SSL certificate .
+
+                :type handle: object
+                :param handle: Object returned from Task Validate
+
+                :type aws_username: str
+                :param aws_username: Username of the IAM user to be looked up
+
+                :rtype: Result Dictionary of result
+    """
+    iamClient = handle.client('iam')
+    paginator = iamClient.get_paginator('list_access_keys')
+    for response in paginator.paginate(UserName= aws_username):
+        for key,value in response.items():
+            if key == "AccessKeyMetadata":
+                return value
```

Comment:
There should be a default return value. If Key is not access key metadata then the Lego will not return anything. This is not a good way of writing Legos.

Reply:
<img width="1287" alt="Screenshot 2022-09-15 at 4 28 10 PM" src="https://user-images.githubusercontent.com/110628398/190862262-ed3a20c9-d436-4da8-b371-87a8568289ea.png">

I was returning only the list of access keys without the response metadata. Will update it to return the entire thing